src/plugins/LADTable/components/LADRow.vue

```diff
@@ -0,0 +1,111 @@
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+<template>
+    <tr>
+        <td>{{name}}</td>
+        <td>{{timestamp}}</td>
+        <td :class="[valueClass]">
+            {{value}}
+        </td>
+    </tr>
+</template>
+
+<style lang="scss">
+
+</style>
+
+<script>
+export default {
+    inject: ['openmct'],
+    props: ['domainObject'],
+    data() {
+        return {
+            name: this.domainObject.name,
+            timestamp: '---',
+            value: '---',
+            valueClass: ''
+        }
+    },
+    methods: {
+        updateValues(datum) {
+            this.timestamp = this.formats[this.timestampKey].format(datum);
+            this.value = this.formats[this.valueKey].format(datum);
+
+            var limit = this.limitEvaluator.evaluate(datum, this.valueMetadata);
+
+            if (limit) {
+                this.valueClass = limit.cssClass;
+            } else {
+                this.valueClass = '';
+            }
+        },
+        updateName(name){
+            this.name = name;
+        },
+        updateTimeSystem(timeSystem) {
+            this.timestampKey = timeSystem.key;
+        }
+    },
+    mounted() {
+        this.metadata = this.openmct.telemetry.getMetadata(this.domainObject);
+        this.formats = this.openmct.telemetry.getFormatMap(this.metadata);
+
+        this.limitEvaluator = openmct
+            .telemetry
+            .limitEvaluator(this.domainObject);
+
+        this.stopWatchingMutation = openmct
+            .objects
+            .observe(
+                this.domainObject,
+                '*',
+                this.updateName
+            );
+
+        this.openmct.time.on('timeSystem', this.updateTimeSystem);
+
+        this.timestampKey = this.openmct.time.timeSystem().key;
+
+        this.valueMetadata = this
+            .metadata
+            .valuesForHints(['range'])[0];
+
+        this.valueKey = this.valueMetadata.key
+
+        this.unsubscribe = this.openmct
+            .telemetry
+            .subscribe(this.domainObject, this.updateValues);
+
+        this.openmct
+            .telemetry
+            .request(this.domainObject, {strategy: 'latest'})
+            .then((values) => values.forEach(this.updateValues));
```

Comment:
considering this can only display a single value, this should only call updateValues once instead of iterating over every value in the response.